### ISSUE TITLE
Review current implementation of status change to make sure it works as stated below

### ISSUE THREAD

**Issue by cdparra** [socialappslab/appcivist#96]
-- Public Drafts (Rascunhos Públicos)
The entire university community can see and comment, but only those who are shared can edit.
-- Published Proposals (Propostas Publicadas)
The entire university community can see and comment, but no one can edit.

After July 1, those who share a private draft can decide to turn it into a public draft or a published proposal.  At any time until August 11, they can change their private drafts into public drafts and vise versa.  After August 11, all drafts must become Published Proposals to advance to next phase.

The change of proposal status in Phase 1 will be done in AppCivist (top nav bar).  In Phase 3, permissions and statuses will be managed in PeerDoc.

Required to change status to either Public Draft or Published Proposal:
-- “Theme” (select 1 from list), “Keywords” (max 3), “Summary” (max 125 words)
Optional:
-- “Campus Proposta” is optional (select one or more from dropdown list)

**Comment by cdparra** [comment]
1. Trying to change to PUBLIC DRAFT and the requirements fail even if the proposal already have the fields that are required. The reason of this is that the backend only checks if the Contribution object has the required fields, but fields like `themes`, `keywords`, and custom fields are not part of the Contribution model. 

related code: 
```
Logger.info("Requirements config: " + requirements);
        List<String> customFields = new ArrayList<>();
        if(!requirements.isEmpty()) {
            for(String requirement: requirements) {
                String output = requirement.substring(0, 1).toUpperCase() + requirement.substring(1);
                try {
                    Contribution.class.getMethod("get" + output);
                } catch (NoSuchMethodException e) {
                    customFields.add(requirement);
                }
            }
```

2. Same happens when trying to change to PUBLISHED

3. Before we can further test, we need to resolve the above
